plugins/modules/cloud/google/gcdns_record.py

```diff
@@ -133,46 +133,46 @@
 '''
 
 EXAMPLES = '''
```

Comment:
Shouldn't this module to move to `google.gcp` ?

Reply:
@gundalow do you know?

Reply:
I don't know, let's fix the doc, then we decide:)

Reply:
maybe they are made not by google

Reply:
Copyright (C) 2015 CallFire Inc.